test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs

```diff
@@ -236,38 +236,37 @@ public void UpdateRunSettingsShouldThrowIfArchitectureSetIsIncompatibleWithCurre
         [DataTestMethod]
         [DataRow("DesignMode")]
         [DataRow("CollectSourceInformation")]
-        public void UpdateRunSettingsShouldNotModifyXmlIfNavigatorIsNotAtRootNode(string settingName)
+        [Ignore]
+        public void UpdateRunSettingsShouldNotModifyXmlIfxmlDocumentIsNotAtRootNode(string settingName)
```

Comment:
Why this is ignored?

Reply:
It is no more required. 
We are using xmldoc and therefore, there is no concept of positioning as it used to be in xpathnavigatoor

Reply:
Got it, Please remove this test.